docs/data/material/components/skeleton/skeleton.md

```diff
@@ -37,7 +37,7 @@ For instance:
 
 ## Variants
 
-The component supports 3 shape variants.
+The component supports 4 shape variants.
```

Comment:
This sounds off. Aren't we mixing aesthetic considerations with the kind of component that the skeleton is meant to replace?

Reply:
> Aren't we mixing aesthetic considerations with the kind of component that the skeleton is meant to replace?

I'm not sure this is related to the change because we already have similar variants like the circle that does the same thing.

> I would expect 2 different props.

I didn't follow you, which two props do you expect? :thinking: